ft/persistence.robot

```diff
@@ -0,0 +1,130 @@
+*** Settings ***
+Resource          common.resource
+Suite Setup       Neuron Context Ready 
+Suite Teardown    Neuron Context Stop
+
+*** Variables ***
+
+*** Test Cases ***
+POST ping, status can be ok
+    Ping
+
+Get the newly added adapters after logout and login back, it should return success
+    Add Node                 type=${${NODE_DRIVER}}    name=modbus-tcp-1       plugin_name=modbus-tcp-plugin
+    Add Node                 type=${${NODE_DRIVER}}    name=modbus-tcp-2       plugin_name=modbus-tcp-plugin
+    Add Node                 type=${${NODE_DRIVER}}    name=modbus-tcp-3       plugin_name=modbus-tcp-plugin
+    Add Node                 type=${${NODE_DRIVER}}    name=modbus-tcp-4       plugin_name=modbus-tcp-plugin
+    ${res} =                 Add Node                  type=${${NODE_MQTT}}    name=mqtt-1                      plugin_name=mqtt-plugin
+    Check Response Status    ${res}                    200
+    Check Error Code         ${res}                    ${ERR_SUCCESS}
+
+    LOGOUT
```

Comment:
You should terminate the neuron process instead of just logout to ensure persistence does work.

Reply:
Yes, wrong idea, I'm modifying.